tests/data/nxdl2yaml/ref_enumerations.yaml

```diff
@@ -0,0 +1,104 @@
+category: base
+doc: |
+  This is a test for enumerations.
+type: group
+NXenums(NXobject):
+  (NXentry):
+    closed_enum_no_docs_list:
+      doc: |
+        This is a closed enumeration without any docstrings for the individual items.
+        The items are stored in a flat list.
+      
```

Comment:
Agree with Rubel that not having this additional line would be cleaner. Having it or not does not change the semantics though.